.github/workflows/build-linux.yml

```diff
@@ -1,6 +1,6 @@
 name: Linux CI
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
This is intentional (to have a small check for pushes, and only do mac/etc on PRs)

Reply:
I like that, too. Also, what happens if I push to an existing PR?

Reply:
Nothing? if develop is not merged?

If you are talking about an existing PR after this, yes these actions will be run twice (but in a good way because one will be the PR build and another the BRANCH build) just like before with Travis.

Reply:
Gotcha. It seemed redundant to have a check for pushed commits that aren't in a PR, but the reasoning is valid.